src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -117,53 +74,22 @@ public boolean isFlashCharts() {
         return flashCharts;
     }
 
-    public void setFlashCharts(boolean flashCharts) {
-        this.flashCharts = flashCharts;
-    }
-
     public boolean isHighCharts() {
         return highCharts;
     }
 
-    public void setHighCharts(boolean highCharts) {
-        this.highCharts = highCharts;
+    public boolean hasTagsReportingEnabled(){
+        return tagsReportingEnabled;
+    }
+
+    public static Builder newReportBuilder(){
+        return new ReportBuilder.Builder();
     }
 
-    /**
```

Comment:
As public constructor is replaced with Builder which has more self-explanatory methods to build this object. Constructor was taking more than 8 arguments which was not really good.